app/views/steps/evidence/upload/_uploaded_file.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr class="govuk-table__row">
+  <td id="<%=document.id%>" class="govuk-table__cell">
+    <span><%= document.filename %></span>
+
+    <% if document.s3_object_key %>
+      <strong class="govuk-tag govuk-tag--green">Uploaded</strong>
+    <% else %>
+      <% document.valid?(:storage) %>
```

Comment:
Error messages are not saved so when documents are retrieved they need to be validated against our criteria so the correct error message can be displayed